styles.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 330px;
```

Comment:
- [ OPTIONAL ] :thinking: Kindly make sure to use percentages as it would make your project responsive and fit all the possible screens :recycle: